pages/linux/ip-route-get.md

```diff
@@ -0,0 +1,24 @@
+# ip route get
+
+> Get a single route - this command gets a single route to a destination and prints its contents exactly as the kernel sees it.
+> More information: <https://manned.org/ip-route>.
+
+- Print route to a destination (1.1.1.1):
+
+`ip route get 1.1.1.1`
+
+- Print route to a destination (1.1.1.1) from source (0.0.0.0):
+
+`ip route get 1.1.1.1 from 0.0.0.0`
+
+- Print route from 127.0.0.2 to 127.0.0.1:
+
+`ip route get 127.0.0.1 from 127.0.0.2`
+
+- Print route from 127.0.0.1 to 0.0.0.0:
+
+`ip route get 0.0.0.0 from 127.0.0.1`
+
+- Print route from 0.0.0.0 to 127.0.0.1:
+
+`ip route get 127.0.0.1 from 0.0.0.0`
```

Comment:
These examples feel repetitive and redundant.

I think they should either be removed, or be replaced with examples that showcase some other potentially useful options (e.g. `iif`, `oif`, `sport`, `dport`)

Reply:
Thanks for the comment. It maybe repetitive in the perspective of the command itself but I chose the combination of addresses that help you get the idea of what routes would work or what kind of paths that user should trace. But I see that it maybe too opinionated.

I'm not familiar with the usages with `iif`, `oif`, `sport`, `dport`. Feel free to suggests a edits or feel free to overwrite with another PR :)